tools/deploy_documentation.sh

```diff
@@ -26,4 +26,4 @@ pwd
 # Push to qiskit.org website
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in tools/rclone.conf.enc -out $RCLONE_CONFIG_PATH -d
 echo "Pushing built docs to website"
-rclone sync --progress ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
+rclone sync --progress --exclude ./locale/** ./docs/_build/html IBMCOS:qiskit-org-web-resources/documentation/machine-learning
```

Comment:
Actually looking at this did you confirm the `./` works. Looking at the exclude regex we use in qiskit/qiskit it's `locale/**`: https://github.com/Qiskit/qiskit/blob/master/tools/other-builds.txt#L1 (which is also what we used prior to the separate exclude file: https://github.com/Qiskit/qiskit/blob/453c57b348b5f0c8fe80493953bf5eb30cec3b04/tools/deploy_documentation.sh#L32

Reply:
I'll push a commit to use `locale/**` just to be on the safer side

Reply:
Fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/165/commits/d81d4845e034ca6e71d0cd6258222a4acc772e51